mlos_core/mlos_core/optimizers/utils.py

```diff
@@ -0,0 +1,36 @@
+import inspect
```

Comment:
```suggestion
#
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT License.
#
"""
Docstring!
"""
import inspect
```
CI should not pass this - @bpkroth do you know why we are not detecting these style violations?